Formula/dvc.rb

```diff
@@ -289,6 +284,9 @@ class Dvc < Formula
     sha256 "a88ee8903aa0a81f6c3cec2d5cf62d3c8aa67c06439b0496b49048fb1854ebf4"
   end
 
+  # no tarbal for 0.7 on pypi
+  # issue tracker, https://github.com/bdcht/grandalf/issues/26
+  # https://github.com/iterative/dvc/blob/f3010d1d65bf2eb181c63f4bf0c3ca01b837f50b/requirements/default.txt#L13-L14
```

Comment:
Comments brew `update-python-resources`, so let's move this before all the `resource`s.